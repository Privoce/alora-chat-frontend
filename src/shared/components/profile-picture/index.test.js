import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	ProfilePictureComponent
} from 'shared/components';

describe('<ProfilePictureComponent />', () => {
	const modelI = {
		color: 'white',
		backgroundColor: 'red',
		label: 'teste',
		width: 60,
		height: 60
	};

	const getWrapper = (model) => {
		return shallow((
			<ProfilePictureComponent {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
