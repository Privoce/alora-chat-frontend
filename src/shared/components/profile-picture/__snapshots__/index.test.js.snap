// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePictureComponent /> renders correctly 1`] = `
<div
  className="profile-picture fadeIn"
  style={
    Object {
      "backgroundColor": "red",
      "color": "white",
      "height": 60,
      "width": 60,
    }
  }
>
  <LabelComponent
    fontBold={true}
    text="TE"
  />
</div>
`;
